mordred/task_panels.py

```diff
@@ -219,7 +219,7 @@ def __create_dashboard(self, panel_file, data_sources=None):
             logger.info("Not creating a panel that exists already: %s", dash_id)
         else:
             logger.info("Creating the panel: %s", dash_id)
-            if data_sources and 'pipermail' in data_sources:
+            if data_sources and ('pipermail' in data_sources or 'hyperkitty' in data_sources):
                 # the dashboard for mbox and pipermail are the same
```

Comment:
the comment should be removed or mention hyperkitty too

Reply:
Fixed! Thanks @valeriocos.

Merging it!